packages/flutter_tools/lib/src/android/migrations/min_sdk_version_migration.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+
+import '../../base/file_system.dart';
+import '../../base/project_migrator.dart';
+import '../../project.dart';
+import '../gradle_utils.dart';
+
+@visibleForTesting
+const String flutterMinSdk = 'minSdkVersion flutter.minSdkVersion';
```

Comment:
nit: flutterMinSdk is not as clear of a variable name as it could be. What do you think about replacementMinSdkText?

Reply:
changed